src/react-components/room/AudioPopoverButton.js

```diff
@@ -18,13 +18,31 @@ export const AudioPopoverButton = ({
   content,
   isMicrophoneMuted,
   isMicrophoneEnabled,
+  micLevel,
   onChangeMicrophoneMuted
 }) => {
   const ref = useRef();
   const intl = useIntl();
   const title = intl.formatMessage(invitePopoverTitle);
   const popoverApiRef = useRef();
 
+  useEffect(
```

Comment:
LGTM my only comment would be to use callback refs instead of just passing down the ref as we could end up with stale refs if the component changes downstream which in this case shouldn't happen but it's a good practice: https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node

Reply:
Swapped the useRef / useEffect for a useCallback

Reply:
Thanks! I want to refactor this in another PR to try to limit the re-renders to the most inner button element.